src/ai/backend/manager/models/base.py

```diff
@@ -208,9 +208,9 @@ def process_bind_param(
             return value.to_json()
         return value
 
-    def process_result_value(self, raw_value: Dict[str, str], dialect) -> ResourceSlot:
+    def process_result_value(self, raw_value: Dict[str, str], dialect) -> ResourceSlot | None:
```

Comment:
This should be `Optional`.
```suggestion
    def process_result_value(self, raw_value: Dict[str, str] | None, dialect) -> ResourceSlot | None:
```